### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Oluwadamilareolusakin**

### CODE REVIEW

**Comment by maelfosso** [comment]
# Under review by Mael FOSSO

File: enumerable.erb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        false
+    end
+
+    def my_none?
+        return true unless block_given?
+        self.my_each do |item|
+            return true unless yield(item) 
+        end
+        false
+    end
```
Comment (maelfosso) [inline enumerable.erb]: Not working. Please, correct your code

```
>> %w{ant bear cat}.none? { |word| word.length >= 4 }
=> false
>> %w{ant bear cat}.my_none? { |word| word.length >= 4 }
=> true
>> 
```

You should return **true** if all the elements survive to the **each loop** and **false** if any of them is successful to the condition.

File: enumerable.erb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        end
+        self.my_each do |el|
+            unless block_given? 
+                base *= el
+            end
+
+            base = yield(base, el)
+        end
+        base
+    end
```
Comment (maelfosso) [inline enumerable.erb]: Not working.  Please, correct your code
```
>> [5,6,7,8,9,10].inject { |sum, n| sum + n } 
=> 45
>> [5,6,7,8,9,10].my_inject { |sum, n| sum + n } 
=> 46
```

**Comment by maelfosso** [comment]
Hello coders,

You did a really nice job. Really nice code, was simple to read. But before moving forward, there are some changes you should do

- [ ] Please, update your README file. Put the project link, the author name (even if it's a solo project)

- [ ] First, please your name file should not be `enumarable.erb` but **enumerable.rb**. Please, change the extension

- [ ] Please, read the comment above.

Kindly make the suggested changes and submit a new code review request.

File: enumerable.erb (hunk 1 above)
Comment (Oluwadamilareolusakin) [reply enumerable.erb]: Noted, I just corrected it.

File: enumerable.erb (hunk 2 above)
Comment (Oluwadamilareolusakin) [reply enumerable.erb]: Corrected

**Comment by maelfosso** [comment]
# Good job

You can now submit a code review request

**Comment by Oluwadamilareolusakin** [comment]
@maelfosso Hi Mael, do you mean I can merge?

**Comment by maelfosso** [comment]
Yes @Oluwadamilareolusakin

**Pull request merged**